### PR TITLE
Add Web Playback SDK to React app to connect to Spotify as a Web Player

### PR DESCRIPTION
**Pull request by 3Ustin**
The SDK Web Playback was added to the react file to connect to Spotify however somethings were left out. Validation of the token key from Spotify needs to be run through the props as well as hidden from user information. Issues will be created for both separately. The 'ts-ignores' are temporary to allow the code to run, and there is an issue placed in order to resolve that potential issue.

closes issue #5

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -39,5 +39,7 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <!-- This is to import the SDK -->
```
Comment (MtuMartian) [inline public/index.html]: We can remove this comment now

**Review by MtuMartian — COMMENTED** [review]
Overall looks good! Let's do some pair programming on unit tests before we merge this

File: public/index.html (hunk 1 above)
Comment (3Ustin) [reply public/index.html]: Just deleted this Comment, and have started working on tests.

**Review by MtuMartian — APPROVED** [review]
Looks good!

As we discussed in our meeting, let's merge this now so that it's not a blocker for you to work on other issues. We can write the tests after we've studied the React Testing Library

**Pull request merged**